modules/cgrindel_bazel_starlib/0.14.4/MODULE.bazel

```diff
@@ -0,0 +1,116 @@
+module(
+    name = "cgrindel_bazel_starlib",
+    version = "0.14.4",
+)
+
+bazel_dep(
+    name = "bazel_skylib_gazelle_plugin",
+    version = "1.4.1",
+    dev_dependency = True,
+)
+
+bazel_dep(
+    name = "rules_go",
+    version = "0.38.1",
+    repo_name = "io_bazel_rules_go",
+)
+bazel_dep(
+    name = "gazelle",
+    version = "0.29.0",
+    repo_name = "bazel_gazelle",
+)
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(
+    name = "stardoc",
+    version = "0.5.3",
+    repo_name = "io_bazel_stardoc",
+)
+bazel_dep(
+    name = "buildifier_prebuilt",
+    version = "6.0.0.1",
+)
+bazel_dep(name = "platforms", version = "0.0.6")
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(name = "go_sdk")
```

Comment:
See my comment on the previous PR.

Reply:
@fmeum Could you approve this for now? I am trying to debug a presubmit failure on the BCR CI machines.

Reply:
Sure, done!